src/views/TradingCompetition/index.tsx

```diff
@@ -99,9 +98,7 @@ const TradingCompetition = () => {
   const [team3LeaderboardInformation, setTeam3LeaderboardInformation] = useState({ teamId: 3, leaderboardData: null })
 
   const isCompetitionLive = currentPhase.state === LIVE
-  const hasCompetitionEnded =
-    currentPhase.state === FINISHED || currentPhase.state === CLAIM || currentPhase.state === OVER
-
+  const hasCompetitionEnded = EndedPhasesSet.has(currentPhase.state);
```

Comment:
Nice. Have not considered this pattern but it is definitely more readable.

Reply:
🎉